clusterloader2/testing/nichotr/config.yaml

```diff
@@ -0,0 +1,143 @@
+#Constants
+{{$NODE_MODE := DefaultParam .NODE_MODE "allnodes"}}
+{{$NODES_PER_NAMESPACE := DefaultParam .NODES_PER_NAMESPACE 1}}
+{{$PODS_PER_NODE := DefaultParam .PODS_PER_NODE 56}}
+{{$ENABLE_CHAOSMONKEY := false}}
```

Comment:
You can probably leave this out since we're not using chaosmonkey.

Reply:
For some reason, it still goes through with chaosmonkey even if I set this to false, this was on my list of TODO.